src/lib/select/_select-theme.scss

```diff
@@ -34,7 +34,7 @@
     }
   }
 
-  .md-select-content {
+  .md-select-panel {
```

Comment:
The main reason was that I wanted to keep it together with the animation logic since it's animation-related. I've changed it, can you take another look @kara?